redaxo/src/core/lib/rex.php

```diff
@@ -131,15 +131,45 @@ public static function setProperty($key, $value)
      * @throws InvalidArgumentException on invalid parameters
      *
      * @return mixed The value for $key or $default if $key cannot be found
+     * @psalm-return (
+     *     $key is 'login' ? rex_backend_login :
+     *     ($key is 'debug' ? bool|array{enabled: bool, throw_always_exception: bool|int} :
```

Comment:
Ich meine, die Property wäre immer ein Array, selbst wenn in der config.yml noch ein bool-wert steht (compat).